deb/libsoletta/control

```diff
@@ -0,0 +1,34 @@
+Source: libsoletta
+Priority: optional
+Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com>
+Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
```

Comment:
libasan0 and libasan1 are missing here. both are installed together with gcc, so testing the pkgs on Ubuntu/debian with gcc hode this issue.